pennylane_lightning/src/tests/CMakeLists.txt

```diff
@@ -2,7 +2,6 @@ cmake_minimum_required(VERSION 3.14)
 
 project(pennylane_lightning_tests)
 
-set(CMAKE_CXX_STANDARD 17)
```

Comment:
I guess this is already pulled in as part of the main library build?

Reply:
As each of all subdirectories (e.g. `tests`, `algorithms`, `simulators` etc.) is not compiled independently, I think I have removed this line to test C++20 features somewhere. Still, I just recovered this line as it does not affect any usual use-case.